packages/react-charting/src/components/AreaChart/AreaChart.types.ts

```diff
@@ -59,6 +59,12 @@ export interface IAreaChartProps extends ICartesianChartProps {
    * Optimize area chart rendering for large data set.
    */
   optimizeLargeData?: boolean;
+
+  /**
+   * Prop to disable shrinking of the chart beyond a certain limit and enable scrolling when the chart overflows
+   * @default false
+   */
+  enableReflow?: boolean;
```

Comment:
Isnt this happening automatically as it is part of IModifiedCartesian props.
We have recently added xAxisTitle, yAxisTitle, rotateXaxisLabels

Reply:
Extended prop types are not visible in the demo site, so I added this for all cartesian charts.

Reply:
Lets revert this for now. I will check why the extended props are not showing on demo site.